charts/README.rst

```diff
@@ -4,6 +4,26 @@ Helm Charts for Deploying RENKU on Kubernetes
 Helm 2.9.0 or newer is recommended as we use the :code:`before-hook-creation` hook deletion policy.
 See also: `before-hook-creation delete policy <https://github.com/kubernetes/helm/commit/1d4883bf3c85ea43ed071dff4e02cc47bb66f44f>`_.
 
+
+Deploying from a Helm repository
+--------------------------------
+
+.. code-block:: console
+
+    $ minikube start
+    $ helm init
+    $ helm repo add renku https://swissdatasciencecenter.github.io/helm-charts/
+    $ helm upgrade --install renku --namespace renku \
```

Comment:
I think this command might need more arguments... 
What we currently have in our [charts README](https://github.com/SwissDataScienceCenter/renku/tree/master/charts#testing-locally) is :
```helm upgrade --install \
    renku ./renku \
    --namespace renku \
    -f minikube-values.yaml \
    --set global.renku.domain=$(minikube ip) \
    --set ui.gitlabUrl=http://$(minikube ip)/gitlab \
    --set ui.jupyterhubUrl=http://$(minikube ip)/jupyterhub \
    --set ui.gatewayUrl=http://$(minikube ip)/api \
    --set gateway.keycloakUrl=http://$(minikube ip) \
    --set gateway.gitlabUrl=http://$(minikube ip)/gitlab \
    --set notebooks.jupyterhub.hub.extraEnv[0].value=http://$(minikube ip)/gitlab \
    --set notebooks.jupyterhub.hub.services.gateway.oauth_redirect_uri=http://$(minikube ip)/api/auth/jupyterhub/token \
    --set notebooks.jupyterhub.auth.gitlab.callbackUrl=http://$(minikube ip)/jupyterhub/hub/oauth_callback \
    --set notebooks.gitlab.registry.host=10.100.123.45:8105 \
    --set gitlab.registry.externalUrl=http://10.100.123.45:8105/ \
    --set graph.gitlab.url=http://$(minikube ip)/gitlab \
    --timeout 1800```

Reply:
it's still in this readme, just lower down.

Reply:
but you're right, we should make this top example very generic, not minikube-specific.

Reply:
great, it's cleaner this way! ✨